tests/system/util.py

```diff
@@ -142,6 +143,27 @@ def mock_datastore_method(self, method: str) -> None:
         orig_method = getattr(DatastoreAdapter, method)
 
         def mock_method(inner_self: DatastoreAdapter, *args: Any, **kwargs: Any) -> Any:
+            if orig_method.__name__ == "get_many":
+                self.calls_detail.append(
+                    (
+                        orig_method.__name__,
+                        {
+                            f"{entry.collection}/{entry.ids}": entry.mapped_fields
+                            for entry in args[0]
+                        },
+                        kwargs.get("lock_result", "undefined"),
+                    )
+                )
+            elif orig_method.__name__ == "get":
+                self.calls_detail.append(
+                    (
+                        orig_method.__name__,
+                        {args[0]: args[1]},
+                        kwargs.get("lock_result", "undefined"),
+                    )
+                )
+            else:
+                self.calls_detail.append((orig_method.__name__, "NotImplemented"))
```

Comment:
For this I had the verbose flag. But if you like this more structured approach, please also print this data below instead of my rudimentary approach (just `print(self.call_details[-1]` should suffice)